WooCommerce/Classes/ViewRelated/Dashboard/Settings/Privacy/PrivacySettingsViewController.swift

```diff
@@ -125,7 +173,7 @@ private extension PrivacySettingsViewController {
         // switch
         cell.isOn = collectInfo
         cell.onChange = { newValue in
```

Comment:
not sure if `[weak self]` would make sense here? the `reportCrashes` diffs also have this :)